omr/omr.cpp

```diff
@@ -139,70 +142,123 @@ int Omr::pagesInDocument() const
 
 bool Omr::readPdf()
       {
+      QProgressDialog *progress = new QProgressDialog(QWidget::tr("Reading PDF..."), QWidget::tr("Cancel"), 0, 100, 0, Qt::FramelessWindowHint);
+      progress->setWindowModality(Qt::ApplicationModal);
+      progress->show();
+      progress->setRange(0, ACTION_NUM);
+
 #ifdef OCR
       if (_ocr == 0)
             _ocr = new Ocr;
       _ocr->init();
 #endif
+      int ID = READ_PDF;
+      int page = 0;
+      bool val;
+      while (ID < ACTION_NUM) {
+            if(ID != INIT_PAGE && ID != SYSTEM_IDENTIFICATION) {
+                  page = 0;
+                  progress->setLabelText(ActionNames.at(ID+1) + " at Page" + QString::number(1));
```

Comment:
This need to be translated and there is a missing space. See also the other line. It should look like 
```
tr("%1 at Page %2").arg(ActionNames.at(ID+1)).arg(1)
```